samza/indexer/src/main/java/org/sunbird/jobs/samza/service/IndexerService.java

```diff
@@ -67,10 +74,33 @@ private void updateES(String type, String key, Message message) {
 
   private Map<String, Object> prepareData(Message message, String type) {
     Map<String, Object> data = null;
-    data = message.getProperties();
+    if (Constants.ORG.equals(type)) {
+      data = prepareOrgData(message.getProperties());
+    } else {
+      data = message.getProperties();
+    }
     return data;
   }
 
+  @SuppressWarnings("unchecked")
+  private Map<String, Object> prepareOrgData(Map<String, Object> map) {
+    String contactDetails = (String) map.get(JsonKey.CONTACT_DETAILS);
```

Comment:
Let us discuss regarding CONTACT_DETAILS. Perhaps, we can drop this from org metadata.

Reply:
okay